OsmAnd/src/net/osmand/plus/plugins/osmedit/dialogs/NewBasicEditPoiFragment.java

```diff
@@ -0,0 +1,268 @@
+package net.osmand.plus.plugins.osmedit.dialogs;
+
+import static net.osmand.plus.plugins.osmedit.dialogs.EditPoiDialogFragment.AMENITY_TEXT_LENGTH;
+import static net.osmand.plus.plugins.osmedit.fragments.EditPoiContentAdapter.TYPE_ADD_OPENING_HOURS;
+import static net.osmand.plus.plugins.osmedit.fragments.EditPoiContentAdapter.TYPE_BASIC_INFO;
+
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.text.InputFilter;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentManager;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import net.osmand.osm.edit.OSMSettings;
+import net.osmand.plus.OsmandApplication;
+import net.osmand.plus.R;
+import net.osmand.plus.activities.MapActivity;
+import net.osmand.plus.base.BaseOsmAndFragment;
+import net.osmand.plus.helpers.LocaleHelper;
+import net.osmand.plus.plugins.osmedit.data.EditPoiData;
+import net.osmand.plus.plugins.osmedit.fragments.EditPoiContentAdapter;
+import net.osmand.plus.utils.AndroidUtils;
+import net.osmand.plus.utils.ColorUtilities;
+import net.osmand.util.OpeningHoursParser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+public class NewBasicEditPoiFragment extends BaseOsmAndFragment implements EditPoiDialogFragment.OnFragmentActivatedListener {
```

Comment:
It looks like there is issue with scroll height 

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/91894f7b-3fcd-4769-9c55-70fba3029b68"  width="250" height="auto"/> </th>
  </tr>
</table>